.gitignore

```diff
@@ -1,10 +1,3 @@
 build/
 .project
-/WebRoot/js/zimbra/
```

Comment:
Okay. No problem. My concern was, `.gitignore` is also helpful to us during development, so that when we create build locally, newly create build and files added during build creation don't shows up untracked files in git log, which is confusing for us. 

Current version of `.gitignore` helps to avoid this by ignoring all files created/added during build time and to make sure that git log only shows actually files which are potential candidate to stage and commit.